### PR TITLE
Break scene into element groups by size

### PR DESCRIPTION
**Pull request by mihhu**

### CODE REVIEW

File: src/packages/excalidraw/components/Collab.tsx [hunk 1]
```diff
@@ -772,8 +772,8 @@ class Collab extends PureComponent<CollabProps, CollabState> {
     this.queueSaveToFirebase();
   };
 
-  queueBroadcastAllElements = throttle(() => {
-    this.portal.broadcastScene(
+  queueBroadcastAllElements = throttle(async () => {
```
Comment (saghul) [inline src/packages/excalidraw/components/Collab.tsx:775]: Why is this change necessary? I didn't see you change the function to be async. Or is this fixing a bug?

File: src/packages/utils.ts [hunk 2]
```diff
@@ -193,6 +193,36 @@ export const exportToClipboard = async (
   }
 };
 
+export function splitArrayByByteSize(
```
Comment (saghul) [inline src/packages/utils.ts:196]: "ByBy"

File: src/excalidraw-app/collab/Portal.tsx [hunk 3]
```diff
@@ -137,46 +138,51 @@ class Portal {
...
-      payload: {
-        elements: syncableElements,
-      },
-    };
-
-    for (const syncableElement of syncableElements) {
-      this.broadcastedElementVersions.set(
-        syncableElement.id,
-        syncableElement.version,
+    const groupedElements = splitArrayByByteSize(allElements, 1000000);
```
Comment (saghul) [inline src/excalidraw-app/collab/Portal.tsx:141]: Did you come up with this fix or is it something cherry-picked from upstream?

Comment (mihhu) [reply src/excalidraw-app/collab/Portal.tsx:141]: I came up with it

File: src/packages/excalidraw/components/Collab.tsx (hunk 1 above)
Comment (mihhu) [reply src/packages/excalidraw/components/Collab.tsx:775]: The method was already async

**Review by saghul — APPROVED** [review]
